src/game/game_init.c

```diff
@@ -88,10 +85,11 @@ u16 sRenderingFramebuffer = 0;
 // Goddard Vblank Function Caller
 void (*gGoddardVblankCallback)(void) = NULL;
 
-// Defined controller slots
-struct Controller *gPlayer1Controller = &gControllers[0];
-struct Controller *gPlayer2Controller = &gControllers[1];
-struct Controller *gPlayer3Controller = &gControllers[2]; // Probably debug only, see note below
+// Defined controller slots. Anything above MAX_NUM_PLAYERS will be unused.
+struct Controller* const gPlayer1Controller = &gControllers[0];
```

Comment:
since player one can change to pointing at any controller, I'd switch these back to not be const